app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -51,81 +46,69 @@ fun PlotSurface() {
         ) {
             Text(
-                text = "COORDINATE PLOTTER",
-                modifier = Modifier.padding(40.dp)
+                text = stringResource(R.string.app_name),
+                modifier = Modifier.padding(10.dp)
             )
+
             Map(xPercent = sliderXPosition, yPercent = sliderYPosition)
+
             SliderX(
                 value = sliderXPosition,
                 valueChanged = { value ->
                     sliderXPosition = value
                 }
             )
+
             SliderY(
                 value = sliderYPosition,
                 valueChanged = { value ->
                     sliderYPosition = value
                 }
             )
+
         }
     }
 }
 
-@Composable
-fun SliderX(value: Float = 0.0f, valueChanged: (Float) -> Unit) {
-    Slider(
-        value = value,
-        onValueChange = valueChanged
-    )
-}
 
-@Composable
-fun SliderY(value: Float = 0.0f, valueChanged: (Float) -> Unit) {
-    Slider(
-        value = value,
-        onValueChange = valueChanged
-    )
-}
-
-private const val MAX_WIDTH = 300
-private const val CIRCLE_DIAMETER = 12
-private const val CIRCLE_RADIUS = CIRCLE_DIAMETER / 2
-
-@Composable
-fun Map(xPercent: Float, yPercent: Float, modifier: Modifier = Modifier) {
-    Box(
-        modifier
-            .background(Color.Blue)
-            .size(300.dp)
-    ) {
-        Box(
-            modifier
-                .offset(
-                    (xPercent * MAX_WIDTH - CIRCLE_RADIUS).dp,
-                    (yPercent * MAX_WIDTH - CIRCLE_RADIUS).dp
-                )
-                .clip(shape = CircleShape)
-                .background(Color.Green)
-                .size(CIRCLE_DIAMETER.dp)
-        )
-    }
-}
-
-@Preview(showBackground = true)
+@Preview(showBackground = true, uiMode = Configuration.UI_MODE_NIGHT_YES)
 @Composable
 fun PlotSurfacePreview() {
     MyApplicationTheme {
         PlotSurface()
     }
 }
 
-@Preview(showBackground = true)
-@Composable
-fun MapPreview() {
-    MyApplicationTheme {
-        Map(xPercent = 0.5f, yPercent = 0.5f)
-    }
-}
+//@Preview(showBackground = true, name = "5-inch Device Portrait", widthDp = 360, heightDp = 640)
+//@Composable
+//fun PlotSurfacePreview() {
+//    MyApplicationTheme {
+//        PlotSurface()
+//    }
+//}
+
+//@Preview(showBackground = true, name = "5-inch Device Landscape", widthDp = 640, heightDp = 360)
+//@Composable
+//fun PlotSurfacePreview() {
+//    MyApplicationTheme {
+//        PlotSurface()
+//    }
+//}
+
+//@Preview(showBackground = true, name = "10-inch Tablet Portrait", widthDp = 600, heightDp = 960)
+//@Composable
+//fun PlotSurfacePreview() {
+//    MyApplicationTheme {
+//        PlotSurface()
+//    }
+//}
+
+//@Preview(showBackground = true, name = "10-inch Tablet Landscape", widthDp = 960, heightDp = 600)
+//@Composable
+//fun PlotSurfacePreview() {
+//    MyApplicationTheme {
+//        PlotSurface()
+//    }
+//}
```

Comment:
These commented out previews could be removed as they're not needed here anymore.

Reply:
yep, thought of removing and then left it just in case needed to again check for different sizes in which case could easily uncomment and check. Will remove this and note it somewhere else. Thank you